src/components/Footer.tsx

```diff
@@ -34,7 +34,7 @@ export const Footer: React.FC<IProps> = () => {
         {footerLinks(FIRST_THREE)}
         <Link href="/">
           <a className="footer-logo">
-            <Image src={BrlLogo.src} alt="BRL-CAD" width={72} height={72} />
+            <Image src={brlLogo.src} alt="BRL-CAD" width={72} height={72} />
```

Comment:
**Totally optional improvement**
I am not sure but I would believe, BRL CAD logo would be used at multiple places.

A more DRY use could be to create a BRLCAD Logo component and then have 3 or 4 standard variants of it , which can be switched by sending a prop.

Reply:
Yeah I have updated this one now we are using a common component with size prop to render logo accordingly